### ISSUE TITLE
Didn't accept my answer

### ISSUE THREAD

**Issue by bricas** [Ovid/app-wordle#1]
Cloned the git repo, tried to play a game, but it wouldn't accept my correct answer.

![image](https://user-images.githubusercontent.com/18400/150006620-68960e18-d36f-4eaf-b6ca-76bb4999ff85.png)

Note, I had to use your example start word cause all of the usual starter words i try wouldn't work (taser, laser, loser...)

**Comment by Ovid** [comment]
Yeah, my word list is far, far too small. `/usr/dict/words` pulls in so many weird things like `aalii` and `zoons` that I went with a much smaller list. 

No clue why the above didn't work, but I confirm that this particular word fails on my machine! Will look into it later.
